src/rpc/common/Types.h

```diff
@@ -65,6 +66,7 @@ struct Context
     // update backend interfaces to use const& yield
     const std::reference_wrapper<boost::asio::yield_context> yield;
     const std::shared_ptr<WsBase> session;
+    const std::shared_ptr<SubscriptionManager> subManager;
```

Comment:
SubscriptionManager just like Backend should be injected thru constructor of the Handler rather than thru Context.